src/extensions/Markdown.js

```diff
@@ -108,7 +108,17 @@ const Markdown = Extension.create({
 						return parser.parseSlice(dom, { preserveWhitespace: true, context: $context })
 					},
 					clipboardTextSerializer: (slice) => {
-						return createMarkdownSerializer(this.editor.schema).serialize(slice.content)
+						const traverseNodes = (slice) => {
+							if (slice.content.childCount > 1) {
+								return createMarkdownSerializer(this.editor.schema).serialize(slice.content)
+							} else if (slice.isLeaf) {
+								return slice.textContent
+							} else {
+								traverseNodes(slice.content.firstChild)
```

Comment:
What scenario are we hitting with that, from a quick look it seems odd that there is no return and we just return the result of `traverseNodes(slice)` in this case still

Reply:
I also think a return is missing here?

Reply:
The intended goal was that, it there is more than one node selected, odds are the user is copying a group of things and would want to retain the markdown. That's why, in the first if block, the markdown serializer is returned. Otherwise, if that is not the case, but the slice is a leaf (the final node in the tree), then it's just a single item and we can copy the raw text and probably don't need the markdown. The else statement is basically like, if neither of those are true, then recurse back into the traverseNodes function to try again on a deeper level of the slice until we find something that can be returned. I hope that makes sense in some way?

Reply:
Yes but you still need to return the output of the recursive `traverseNodes` otherwise in this case `undefined` is returned

Reply:
> Yes but you still need to return the output of the recursive `traverseNodes` otherwise in this case `undefined` is returned

I see, okay, thank you